src/bugsnag.js

```diff
@@ -29,6 +29,18 @@
     // is configurable via the `maxDepth` setting.
     maxPayloadDepth = 5;
 
+  var stacktraceName;
+  try {
+    throw Error('');
+  } catch(err) {
+    stacktraceName = (
```

Comment:
Some comment on which browsers use which error property for the stack trace would be nice.